app/assets/stylesheets/components/_dashboard-teas.scss

```diff
@@ -86,6 +86,10 @@ These are styles related to tea time displays on attendee or host dashboards.
 
     }
 
+    p.description {
+      margin-bottom: 0;
```

Comment:
this kind of thing probably belongs higher level and not nested in such a deep hierarchy. just feedback, i know our styles are messy right now.